docker-compose.dev.yml

```diff
@@ -5,6 +5,12 @@ version: "3.1"
 # * Definitely want to have different env var files for prod vs dev
 
 services:
+  server:
+    command: npm run dev
+    volumes:
+      - "./server:/app"
+    ports:
+      - "9229:9229"
```

Comment:
Ditto for this one. Even just a comment is helpful

Reply:
This will be merged in with the `docker-compose.dev.yml` already in the project. The idea is to pull out all of the "developer conveniences" into a single config overlay, so that the base `docker-compose.yml` can be as close to production ready as possible.

I _think_ this is already documented in one of the READMEs somewhere, but could probably stand to be made more explicit in more places as this side of the stack settles out.

All that having been said, this specific set of changes fires up the server with live code reloading.